packages/core/src/checkbox/checkbox-group.ts

```diff
@@ -12,9 +13,17 @@ import {
 } from '../utils/controllers/index.js';
 import { HintMixin } from '../utils/mixins/index.js';
 import { requiredValidator } from '../utils/index.js';
+import { Checkbox } from './checkbox.js';
 import styles from './checkbox-group.scss.js';
 
-export class CheckboxGroup extends HintMixin(LitElement) {
+export class CheckboxGroup extends HintMixin(ScopedElementsMixin(LitElement)) {
+  /** @private */
+  static get scopedElements(): ScopedElementsMap {
+    return {
+      'sl-checkbox': Checkbox
```

Comment:
`scopedElements` only refer to the custom elements that are rendered by *this* component (in the `render()` method). The checkbox isn't one of those right?